src/components/dialog-test.js

```diff
@@ -411,6 +411,88 @@ describes.realWin('Dialog', (env) => {
       expect(styleDuringInit).to.equal('display: none !important;');
     });
 
+    describe('fade background', () => {
+      it('should not suppress click events by default', async () => {
+        let wasClicked = false;
+        const clickFun = function () {
+          wasClicked = true;
+        };
```

Comment:
Can you use sandbox.spy() for these functions?

Reply:
Any usage of spy seemed to break event registration so the click button no longer worked.